README.md

```diff
@@ -10,6 +10,14 @@ Install the NPM package and save it to your project using
 npm install --save clever-components
 ```
 
+Note: This library uses ES6 features not supported in Internet Explorer such as [`String.startsWith`](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/String/startsWith). To support use of this library in IE, include `babel-polyfill` in the entrypoint to your application. See the [docs](https://babeljs.io/docs/usage/polyfill/).
```

Comment:
ugh. is that really true?
good to know. we should rip those out and use lodash then.